lib/internal/backend/repository/repository2.go

```diff
@@ -167,48 +175,146 @@ func (rb *RepositoryBackend) buildACIV2(layerIDs []string, dockerURL *types.Pars
 	return aciLayerPaths, aciManifests, nil
 }
 
-func (rb *RepositoryBackend) getManifestV2(dockerURL *types.ParsedDockerURL) (*v2Manifest, []string, error) {
+func (rb *RepositoryBackend) buildACIV22(layerIDs []string, dockerURL *types.ParsedDockerURL, outputDir string, tmpBaseDir string, compression common.Compression) ([]string, []*schema.ImageManifest, error) {
+	layerFiles := make([]*os.File, len(layerIDs))
+
+	tmpParentDir, err := ioutil.TempDir(tmpBaseDir, "docker2aci-")
+	if err != nil {
+		return nil, nil, err
+	}
+	defer os.RemoveAll(tmpParentDir)
+
+	copier := progressutil.NewCopyProgressPrinter()
+
+	var errChannels []chan error
```

Comment:
any reason why we cannot use a single (buffered) channel). The amount of err's is known in forehand.